### PR TITLE
Fix any type

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
fixes: https://github.com/EveripediaNetwork/issues/issues/134
fixes https://github.com/EveripediaNetwork/issues/issues/130

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
@JustinCast, kindly review.

**Review by JustinCast — APPROVED** [review]

File: src/components/Elements/ImageInput/ImageInput.tsx [hunk 1]
```diff
@@ -23,11 +21,15 @@ const ImageInput = ({
...
-      responseType: 'arraybuffer',
-    })
-    setImage(new buffer.Buffer(data as Buffer))
+    try {
+      const { data } = await axios.get(String(event.target.value), {
+        responseType: 'arraybuffer',
+      })
+      setImage(new buffer.Buffer(data as Buffer))
+    } catch (error) {
+      return null
```
[the remark this answers is not in the document]
Comment (Emmanueldmlr) [reply src/components/Elements/ImageInput/ImageInput.tsx:30]: That will be good. We can show the alert if the Image is successfully uploaded or not.
Thanks.

[the remark this answers is not in the document]
Comment (Emmanueldmlr) [reply src/components/Elements/ImageInput/ImageInput.tsx]: @JustinCast, We have the notification now.

**Review by JustinCast — APPROVED** [review]
Nice job!

**Comment by JustinCast** [comment]
@Emmanueldmlr the only thing remaining is to solve conflicts

**Comment by JustinCast** [comment]
@Emmanueldmlr the only thing remaining is to solve conflicts

**Comment by JustinCast** [comment]
@Emmanueldmlr the only thing remaining is to solve conflicts

File: .env.example [hunk 2]
```diff
@@ -1,6 +1,9 @@
 PINATA_KEY=1234
 PINATA_SECRET=1234
 PROD=false
-
 NEXT_PUBLIC_INFURA_ID=1234
 NEXT_PUBLIC_WIKI_CONTRACT_ADDRESS=0x9332ad5290cf8de41107712ef408eb0c47dcb057
+NEXT_PUBLIC_POLYGON_MAINNET_CHAIN_ID=137
```
Comment (kesar) [inline .env.example:6]: those doesnt need to be env variables since they are not gonna change. make them in a constant file somewhere

Comment (Emmanueldmlr) [reply .env.example:6]: okay then.

**Review by kesar — APPROVED** [review]

**Pull request merged**